src/options/options.css

```diff
@@ -19,14 +19,16 @@ body.mobile .mobile-incompatible {
   opacity: 0.4;
 }
 
-form {
-  margin: 16px 32px;
-}
-
 section {
   margin-bottom: 32px;
 }
 
+hr {
```

Comment:
Yeah...

Just one thing here:
As you have now copy & pasted the code from teh h1 tag, you have code duplication. This is not good.

It may be better if this can be changed...
Okay, I see the CSS tags are different, but possibly... (just would be nice, I totally see there is some difference here.)